insights/parsers/nproc.py

```diff
@@ -1,6 +1,9 @@
 """
 limits : Files ``limits.conf`` and ``*-nproc.conf``
 ===================================================
+.. warning::
+    Deprecated parser, please use the :class:`AllLimitsConf` combiner instead.
```

Comment:
@xiangce to reference the combiner use the full path, `insights.combiners.limits_conf.AllLimitsConf`, so that the link is functional in the documentation.